services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2423,16 +2423,18 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
           && clusterConfig.isActiveActiveReplicationEnabledAsDefaultForHybrid()) {
         setStore.activeActiveReplicationEnabled = true;
         updatedConfigsList.add(ACTIVE_ACTIVE_REPLICATION_ENABLED);
-        if (!hybridDataReplicationPolicy.isPresent()) {
-          LOGGER.info(
-              "Data replication policy was not explicitly set when converting store to hybrid store: {}."
-                  + " Setting it to active-active replication policy.",
-              storeName);
+      }
 
-          updatedHybridStoreConfig.setDataReplicationPolicy(DataReplicationPolicy.ACTIVE_ACTIVE);
-          updatedConfigsList.add(DATA_REPLICATION_POLICY);
-        }
+      if (setStore.activeActiveReplicationEnabled && !hybridBufferReplayPolicy.isPresent()) {
```

Comment:
Why change the condition from `!hybridDataReplicationPolicy.isPresent()` to `!hybridBufferReplayPolicy.isPresent()`? I don't understand this part; was it a bug? Could you please share some details in the PR description?

Reply:
That was mistake. Updated the commit message. Thanks!